x-pack/test/security_solution_cypress/cypress/e2e/investigations/timelines/esql/discover_timeline_state_integration.cy.ts

```diff
@@ -212,6 +203,7 @@ describe.skip(
         cy.wait(`@${TIMELINE_REQ_WITH_SAVED_SEARCH}`);
         openKibanaNavigation();
         navigateFromKibanaCollapsibleTo(STACK_MANAGEMENT_PAGE);
+        confirmLeaveUnsavedTimeline();
         cy.get(LOADING_INDICATOR).should('not.exist');
         goToSavedObjectSettings();
```

Comment:
there is a chance we could remove line 11 and 12 of this [goToSavedObjectSettings](https://github.com/michaelolo24/kibana/blob/unskip-discover-state-tests/x-pack/test/security_solution_cypress/cypress/tasks/stack_management.ts#L10) no?

Reply:
Yep, will do